pkgs/development/libraries/quarto/default.nix

```diff
@@ -11,14 +11,18 @@
 , makeWrapper
 , python3
 , extraPythonPackages ? ps: with ps; []
+
+# Allow overriding the Python and R used by Quarto
+, QUARTO_PYTHON ? "${python3.withPackages (ps: with ps; [ jupyter ipython ] ++ (extraPythonPackages ps))}/bin/python3"
```

Comment:
This function should probably accept a pythonWithPackages rather than a string and then `preFixup` can use `.interpreter` to get the rest of the path.